lib/protocol/network.js

```diff
@@ -323,6 +323,17 @@ class Network {
     return Network.by(hrp, cmpBech32, network, 'bech32 address');
   }
 
+    /**
+   * Get a network by its bech32m address prefix.
+   * @param {String} hrp
+   * @param {Network?} network
+   * @returns {Network}
+   */
+
+     static fromBech32m(hrp, network) {
+      return Network.by(hrp, cmpBech32m, network, 'bech32m address');
+    }
+
```

Comment:
I don't think this or the `cmpBech32m()` functions are necessary. All we're really doing here is checking the address prefix which (for mainnet) is `bc1` for both bech32 and bech32m.

Maybe we can just clarify the error in the existing function?

```js
  static fromBech32(hrp, network) {
    return Network.by(hrp, cmpBech32, network, 'bech32(m) address');
  }
```

Reply:
Sure, I just thought maybe bech32m is the final implementation of address protocols, So it will be better to write this function to avoid confusion.